src/index.ts

```diff
@@ -367,13 +382,13 @@ export const plugin: Plugin<[FlexibleContainerOptions?], Root> = (options) => {
    * if the paragraph node has more than one child,
    * control whether the node's last child has end marker ":::" or not (check completeness)
    */
-  function analyzeChildren(node: Paragraph): AnalyzeResult {
+  function analyzeChildren(node: Paragraph, fenceLen: number): AnalyzeResult {
```

Comment:
will receive fence instead of fenceLen, according to proposal of getOpeningFence